apps/scan/backend/test/helpers/shared_helpers.ts

```diff
@@ -55,36 +55,31 @@ export async function waitForStatus(
 /**
  * configureApp is a testing convenience function that handles some common configuration of the VxScan app.
  * @param apiClient - a VxScan API client
+ * @param mockAuth - a mock InsertedSmartCardAuthApi
  * @param mockUsbDrive - a mock USB drive
  * @param options - an object containing optional arguments
- * @param options.mockAuth - a mock InsertedSmartCardAuthApi. Passing this will automatically
- *                           create a mock that auths the user as an election manager of the same
- *                           election defined in the ballot package.
  */
 export async function configureApp(
   apiClient: grout.Client<Api>,
+  mockAuth: InsertedSmartCardAuthApi,
```

Comment:
Every consumer of this function specifies a `mockAuth` because auth is now required for configuration. So I've gone ahead and made a `mockAuth` a required input